frontend/apps/reader/modules/readergesture.lua

```diff
@@ -75,9 +75,9 @@ local action_strings = {
     toggle_gsensor = _("Toggle accelerometer"),
     toggle_rotation = _("Toggle rotation"),
 
-    wifi_on = _("Enable wifi"),
-    wifi_off = _("Disable wifi"),
-    toggle_wifi = _("Toggle wifi"),
+    wifi_on = _("Turn on Wi-Fi"),
+    wifi_off = _("Tun off Wi-Fi"),
```

Comment:
It would be preferable to do the reverse. That is, change every accidental occurrence of "Wi-Fi" to wifi.

(Also, major typo. ;-) )

Reply:
But I kind of *like* Wi-Fi! ;p.

(In fact, most of those occurrences are probably from me ;p).

Reply:
I'm likely responsible for a decent amount of wifi, but it wasn't a mistake. I'm well aware of what the Wi-Fi Alliance calls itself, but in my usage hifi and wifi are fundamentally the same. That applies regardless whether we're talking about it as an abbreviation of a phrase like wireless fidelity or as a generic concept of wireless network, free from the confines of any particular trademark, patent, or technical standard.

(I recognize that in well-edited English prose, wifi is still in a minority. This will change.)

Reply:
If you read the actual claims of the Wi-Fi Alliance, it only concerns selling a physical product that makes use of Wi-Fi technology. By any stretch of trademark law, other uses are unenforceable, and especially now that "Wi-Fi" is a word. As it will remain being outside of whatever possibly consistent action is taken in KOReader. "wifi" just looks wrong, and pollutes the otherwise "Wi-Fi" translation memory of Weblate. Alternatives are "WLAN" and "wireless network".

Reply:
No one said anything about the claims of the Wi-Fi Alliance. What I wrote is that no matter if you look at it as an abbreviation of wireless fidelity or as a generic descriptor, the correct spelling is _wifi_.

WLAN is something like Central European, not exactly regular English. It can be an excellent translation for wifi/Wi-Fi. Wireless network is a bit verbose and possibly goofy, but better.

As for any "pollution," such loaded language makes me want to take up a principled stance against Wi-Fi pollution. :imp: But seriously, a translation memory can't be "polluted" by a slightly different spelling. Next thing you know someone will spell _color_ like _colour_. Oh, the humanity…

Reply:
I grant you that point, and it is one of semantics. Apply it to that of "Wi-Fi*", and you will find nobody spells it _wifi".

https://dictionary.cambridge.org/dictionary/english/wi-fi?q=wifi
https://www.dictionary.com/browse/wifi?s=t
https://dictionary.cambridge.org/dictionary/english/wi-fi?q=wifi
https://www.vocabulary.com/dictionary/WiFi https://en.wikipedia.org/wiki/Wi-Fi

Meanwhile, spelling it "Wi-Fi", _isn't wrong_, _and_ enjoys the most popularity.

Some readers have 3G/WiMAX/4G and so on too. My personal preference is to use something more generic when it is possible. Even https://emojipedia.org/antenna-with-bars/ would do.

Reply:
Nobody except for Nerdist, Business Insider, Gizmodo, Techcrunch, Public Contract Law Journal, Analog, Literary Review, Daily Beast, MotherJones, NewStatesMan, Denver, CSMonitor, and FantasySciFi, just to name a few. Many other publications use WiFi, Wifi, as well as wi-fi. TIME Magazine switched from Wi-Fi to wi-fi in 2003 when it turned generic.

Note that I [said](https://github.com/koreader/koreader/pull/5746#discussion_r363202383) I want to be part of the change:

> I recognize that in well-edited English prose, wifi is still in a minority. This will change.

I assume you included the Cambridge Dictionary twice to emphasize that it does, in fact, fully agree with me. ;-) The fact that it happens to spell the words in question _hi-fi_ and _wi-fi_ is not the salient point, only that it treats them the same.

PS I thought the corpus of US Supreme Court Opinions was quite amusing.

https://caselaw.findlaw.com/us-supreme-court/13-896.html
> Suppose an extensive business headquarters or a resort or a college campus wants a single, central wireless system (sometimes called a Wi-Fi network). In order to cover the large space, the system needs multiple base stations so a user can move around the area and still stay connected.

Reply:
@Frenzie You will never see anything but "Wi-Fi" from the Wi-Fi alliance. This ensures anything but "Wi-Fi", which is in clear minority for good reason, will always be and look like a knock-off.
My changes improve consistency within KOReader, and also help avoid errors in translation, as anything but "Wi-Fi" isn't allowed elsewhere.
Using "wifi" for wireless network is an erroneous practice.

Reply:
@comradekingu https://github.com/koreader/koreader/pull/5746#issuecomment-571270197

> Anyway, let's just drop the changes to the comments & keep the rest of the PR as is (with turn on/off and Wi-Fi).